test/preset-wind-targets.ts

```diff
@@ -1,12 +1,86 @@
-
 export const presetWindiTargets: string[] = [
-  // tables
-  'border-collapse',
-  'border-separate',
-  // object-position
-  'object-center',
-  'object-ct',
-  'object-cb',
+  // animation
+  'animate-none',
+  'animate-pulse',
+  'animate-pulse-alt',
+  'animate-300',
+  'animate-100s',
+  'animate-duration-100',
+  'animate-duration-100.32',
```

Comment:
Could you have the refactoring in another PR so we can have a clearer state? Thanks